__mocks__/@thecointech/rbcapi.ts

```diff
@@ -1,29 +1,21 @@
 import { ETransferErrorCode, RbcApi as SrcApi } from "@thecointech/rbcapi";
-
 export { ETransferErrorCode };
 
-let results = [
-  ETransferErrorCode.Success,
-  ETransferErrorCode.AlreadyDeposited,
-  ETransferErrorCode.InvalidInput,
-  ETransferErrorCode.Cancelled,
-  ETransferErrorCode.UnknownError
-];
-let result = 0;
+let result = 1234;
 export class RbcApi implements Pick<SrcApi, keyof SrcApi> {
-  depositETransfer() {
-    const code = results[result % results.length];
+  depositETransfer = jest.fn(() => {
     result += 1;
+    // Always succeeds (but can be mocked for failure if necessary)
     return Promise.resolve({
-      message: code.toString(),
-      code,
-      confirmation: code == ETransferErrorCode.Success ? result : undefined,
+      message:  ETransferErrorCode.Success.toString(),
+      code:  ETransferErrorCode.Success,
+      confirmation: result += 1,
```

Comment:
Our mocked bank randomly failed transactions. It was a lazy way to run tests over more code, but it's not particularily useful because the outcomes aren't known in advance.  Also, it made the mock unusable for dev work.

By using jest mock fns, we can delegate control of the return value to the testing code, and reuse the mock inside our dev/prodtest systems.